third_party/3/pandas/_typing.pyi

```diff
@@ -90,7 +90,12 @@ Function = Union[np.func, Callable[..., Any]]
 # Also including function names e.g. np.exp. 'sqrt'
 FunctionOrName = Union[Function, str]
 # Used in SelectionMixin and inheriting classes
-AggregationFunction = Union[FunctionOrName, List[FunctionOrName], Dict[Axis, Union[FunctionOrName, List[FunctionOrName]]]]
+MultipleAggregationFunctions = Union[
```

Comment:
Instead of using Union with None, could we use Optional in the method signature?